src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor_tab.cljs

```diff
@@ -0,0 +1,133 @@
+    [org.broadinstitute.firecloud-ui.common.table :as table]
+    [org.broadinstitute.firecloud-ui.paths :as paths]
+    [org.broadinstitute.firecloud-ui.utils :as utils]))
+
+
+(defn- create-mock-submission-details [submission-id]
+  {})
+
+
+(react/defc SubmissionDetails
+  {:render
+   (fn [{:keys [props state]}]
+     (let [server-response (:server-response @state)
+           {:keys [submissions error-message]} server-response]
+       (cond
+         (nil? server-response)
+         [:div {:style {:textAlign "center"}} [comps/Spinner {:text "Loading analysis details..."}]]
+         error-message (style/create-server-error-message error-message)
+         :else
+         [:div {} "id: " (:submission-id props)])))
+   :component-did-mount
+   (fn [{:keys [props state]}]
+     (let [url (paths/submission-details (:workspace-id props) (:submission-id props))
+           on-done (fn [{:keys [success? status-text get-parsed-response]}]
+                     (swap! state assoc :server-response (if success?
+                                                           {:submissions (get-parsed-response)}
+                                                           {:error-message status-text})))
+           canned-response {:responseText (utils/->json-string
+                                           (create-mock-submission-details (:submission-id props)))
+                            :status 200
+                            :delay-ms (rand-int 2000)}]
+       (utils/ajax-orch url {:on-done on-done :canned-response canned-response})))})
+
+
+(defn- create-mock-submissions-list [workspace-id]
+  (map
+    (fn [i]
+      {:workspaceName workspace-id
+       :methodConfigurationNamespace "my_test_configs"
+       :submissionDate (str "2015-" (rand-nth (range 1 12)) "-"
+                         (rand-nth (range 1 30)) "T19:08:53.027Z")
+       :submissionId "46bfd579-b1d7-4f92-aab0-e44dd092b52a"
+       :notstarted []
+       :workflows [{:messages []
+                    :workspaceName workspace-id
+                    :statusLastChangedDate (str "2015-" (rand-nth (range 1 12))
+                                             "-" (rand-nth (range 1 30))
+                                             "T19:08:53.027Z")
+                    :workflowEntity {:entityType "sample"
+                                     :entityName "sample_01"}
+                    :status "Succeeded"
+                    :workflowId "97adf170-ee40-40a5-9539-76b72802e124"}]
+       :methodConfigurationName (str "test_config" (inc i))
+       :status (rand-nth ["Running" "Failed" "Completed" "Queued"])
+       :submissionEntity {:entityType "sample"
+                          :entityName (str "sample_" (inc i))}
+       :submitter "abaumann@broadinstitute.org"})
+    (range (rand-int 50))))
+
+
+(defn- render-submissions-table [submissions on-submission-clicked]
+  [table/Table
+   {:columns
+    [{:header "Date" :starting-width 200 :sort-by :value
```

Comment:
Thanks for catching that. You were right on. Added sort-by and filter-by functions.